spec/draft-barnes-cfrg-4q.md

```diff
@@ -109,29 +112,35 @@ N = 0x29cbc14e5e0a72f05397829cbc14e5dfbd004dfe0f79992fb2540ec7768ce7
 
 # Curve Points
 
-Elements of GF(p) are represented as 16-octet little-endian integers.  An element
-x0 + x1*i of GF(p^2) is represented on the wire by the concatenation of the byte strings for
-x0 and x1. Implementations will use whatever internal representation they desire, but we will
-describe the operations on elements of GF(p^2) assuming that X=x0+x1*i, with each x0 and x1
-an element of GF(p).
+Elements a = (a0, a1, ... , a126)_2 in GF(p) are represented as 128-bit (16-octet) little-endian 
+integers by concatenating a || 0, i.e., elements of GF(p) are encoded as (a0, a1, ... , a126, 0)_2. 
```

Comment:
Well, an "octet" is by definition 8 bits :)  And I wouldn't worry about platforms where a byte isn't 8 bits.

That said, I agree that this is not a very useful encoding.  Changing byte-level endianness is common; changing bit-level endianness is a huge pain.  I think people understand that "little-endian integer" means, but if we want to be more explicit here, we could just steal from RFC 7748:

```
   The u-coordinates are elements of the underlying field GF(2^255 - 19)
   or GF(2^448 - 2^224 - 1) and are encoded as an array of bytes, u, in
   little-endian order such that u[0] + 256*u[1] + 256^2*u[2] + ... +
   256^(n-1)*u[n-1] is congruent to the value modulo p and u[n-1] is
   minimal.
```